docs/list.yaml

```diff
@@ -50,6 +50,42 @@ PHP:
       description: Adds CORS headers support in your Laravel application
       url: https://github.com/barryvdh/laravel-cors
       tags: ['Laravel', 'CORS']
+    barryvdh/laravel-debugbar:
+      description: Laravel Debugbar (Integrates PHP Debug Bar)
+      url: https://github.com/barryvdh/laravel-debugbar
+      tags: ['Laravel', 'Debugging']
+    jenssegers/mongodb:
+      description: A MongoDB based Eloquent model and Query builder for Laravel (Moloquent)
+      url: https://github.com/jenssegers/laravel-mongodb
+      tags: ['Database', 'Moloquent']
+    robmorgan/phinx:
+      description: Simple PHP Database Migrations
+      url: https://github.com/cakephp/phinx
+      tags: ['Migrations']
+    phpoffice/phpspreadsheet:
+      description: A pure PHP library for reading and writing spreadsheet files
+      url: https://github.com/jenssegers/laravel-mongodb
+      tags: ['Office']
+    spatie/laravel-activitylog:
+      description: Log activity inside your Laravel app https://docs.spatie.be/laravel-activitylog
+      url: https://github.com/spatie/laravel-activitylog
+      tags: ['Laravel', 'Activity Log', 'Logging']
+    vinkla/laravel-hashids:
+      description: A Hashids bridge for Laravel (generate short unique ids from integers)
+      url: https://github.com/vinkla/laravel-hashids
+      tags: ['Laravel', 'Hashids']
+    rap2hpoutre/laravel-log-viewer:
+      description: Log Viewer for Laravel 5 (compatible with 4.2 too) and Lumen
+      url: https://github.com/rap2hpoutre/laravel-log-viewer
+      tags: ['Laravel', 'Logging', 'Log Viewer']
+    arcanedev/log-viewer:
+      description: Provides a log viewer for Laravel 5
+      url: https://github.com/ARCANEDEV/LogViewer
+      tags: ['Laravel', 'Logging', 'Log Viewer']
+    hidehalo/nanoid-php:
```

Comment:
For what reason should we prefer IDs, generated by this package over [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier) v4?
The only advantage - shorter string length is very week, IMHO.

While UUIDs:
* Industry-wide standard: [RFC 4122](https://tools.ietf.org/html/rfc4122)
* Supported on multiple platrorms and even as data type in DB engines (Postgres, MS SQL) - they are stored there not as string, but as 128-bit number, which is much more compact, than 21-char string and quicker to process.
* URL-friendly as well

Reply:
@populov I think we could use this package in some cases:
- Generate short codes for "Url Shorten" feature 
- Generate a One Time Password (OTP)
As we know, there are many 3rd parties provide "Url Shorten" service, but use their domain.
So, if we want to use the application domain, we must implement it ourselves and this package may be a good choice

Reply:
For ULS shortener [vinkla/laravel-hashids](https://github.com/vinkla/laravel-hashids) would be much more relevant - just insert record in db, get BIGINT autoincrement ID, assigned to this record by DB engine, and shorten it with _hashids_.

21 chars is too long for manually entered OTP. As for just tokens, that are not supposed to be entered manually, [random_bytes()](http://php.net/manual/en/function.random-bytes.php) will do (except URL friendly).

For unique, short, URL-friendly strings we can just generate UUIDv4, treat it as int and encode with _hashids_ (see [Representing a UUID as a Base62 Hash ID for short, pretty URLs](https://medium.com/@huntie/representing-a-uuid-as-a-base-62-hash-id-for-short-pretty-urls-c30e66bf35f9)).

Just one string
```php
Hashids::encode((int) base_convert(str_replace('-', '', Uuid::uuid4()->toString()), 16, 10));
```
is equivalent of the whole this package.

We still can use it, but I don't see a great value in it, worthing to add to list of recommended packages.

Reply:
Agree, I will remove it